apps/ui/lev-help.md

```diff
@@ -0,0 +1,72 @@
+# Help with LEV
+
+If you spot a problem with any page or function of this website, please let us
+know.
+
+The LEV service is available 24/7, with support provided during business hours
+(8am to 6pm, Monday to Friday).
+
+## Reporting a problem with this website
+### Initial report
+If there is a technical issue preventing you from using the website, HMPO users
+must raise an incident via the ITNow Service Desk, or via the HMPO Service Desk
+for those HMPO End Users who are using residual AdminIT.
```

Comment:
Residual AdminIT - is that a commonly used phrase? Sounds a bit weird to me. Same with End Users.